src/hive_mind_bridge/test/utils/Logger.h

```diff
@@ -0,0 +1,69 @@
+#ifndef HIVE_MIND_BRIDGE_LOGGER_H
+#define HIVE_MIND_BRIDGE_LOGGER_H
+
+#include <cpp-common/ILogger.h>
+#include <cstdarg>
+#include <cstdio>
+#include <string>
+
+class Logger : public ILogger {
+  public:
+    Logger() {}
+
+    LogRet log(LogLevel level, const char* format, ...) override {
```

Comment:
I don't think you actually need the logger logic for your tests, you could just ignore all args and always return ok. Printing during tests is not recommended

Reply:
Right. The thing is, some of my integration tests are run manually for now and I need the logging, so I'll just keep this as-is for now.